kili/queries/asset/__init__.py

```diff
@@ -132,6 +139,24 @@ def assets(self,
             - `metadata_where = {key2: [2, 10]}` to filter on assets whose metadata
                 have key "key2" with a value between 2 and 10.
 
+        !!! example "How to filter based on label categories"
+            The search query is composed of logical expressions following this format:
+
+                [job_name].[category_name].count [comparaison_operator] [value]
+            where:
+
+            - `[job_name]` is the name of the job in the interface
+            - `[category_name]` is the name of the category in the interface for this job
+            - `[comparaison_operator]` can be one of: [`==`, `>=`, `<=`, `<`, `>`]
+            - `[value]` is an integer that represents the count of such objects of the given category in the label
+
+            These operations can be separated by OR and AND operators
+
+            Example:
+
```

Comment:
Add a comment for each line (especially to show that the classification case is handled)